RELEASE.txt

```diff
@@ -29,6 +29,8 @@ CHANGED/ENHANCED EXISTING FUNCTIONALITY
       It will now allow callables with expected args, and any extra args as long as they
       have default arguments. Additionally functions with no defaults for extra arguments
       as long as they are set using functools.partial to create a new callable which set them.
+    - Internal has_key methods removed from SCons' dictionary-like objects,
```

Comment:
Can you list which objects are affected?

Reply:
Should be SubstitutionEnvironment and OverrideEnvironment, I can check later. The others all all in mocked up classes in unittests, I don't think those are worth mentioning?

Reply:
Sure. Only the "real" classes worth noting. Just so it's "easier" to find if they update and something breaks.

Reply:
Added.